app/Models/Campaign.php

```diff
@@ -21,7 +21,7 @@ class Campaign extends Model
      *
      * @var array
      */
-    protected $fillable = ['internal_title', 'cause', 'start_date', 'end_date'];
+    protected $fillable = ['id','internal_title', 'cause', 'secondary_causes', 'campaign_run_id', 'impact_doc', 'start_date', 'end_date', 'created_at', 'updated_at'];
```

Comment:
Let's remove `id`, `created_at`, and `updated_at` from the fillable array & use `Campaign::forceCreate` in the script (since we don't want these to be mass-assignable elsewhere).

Reply:
cool i didn't know about `forceCreate` ! Much better solution!